roles/tas_single_node/tasks/certificates.yml

```diff
@@ -1130,12 +1142,14 @@
         src: "{{ tas_single_node_tsa_signer_private_key }}"
       register: old_key_content
       when: (certs_dir_files.files | selectattr('path', 'equalto', tas_single_node_tsa_signer_private_key) | list | length) != 0
+      no_log: true
 
     - name: Read the old cert chain content
       ansible.builtin.slurp:
         src: "{{ tas_single_node_tsa_certificate_chain }}"
       register: old_cert_chain_content
       when: (certs_dir_files.files | selectattr('path', 'equalto', tas_single_node_tsa_certificate_chain) | list | length) != 0
+      no_log: true
```

Comment:
Certificate chain is not sensitive content

Reply:
@jourdee-lab You closed this but didnt fulfill the request